searchbar.html

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <!-- Font awesome Icon CDN -->
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.3.0/css/all.min.css"
+    />
+    <link rel="stylesheet" href="searchbar.css" />
+    <title>Dropdown Search Bar | upCoding</title>
+  </head>
+  <body>
+
+        <!--Search Bar Start-->
```

Comment:
I assume this is a test file -- since you have several of them, would you mind moving them all into a folder like `/test`? that'll just make the code a bit more clear.